api/error_handlers/errors.js

```diff
@@ -0,0 +1,4 @@
+exports.handleServerError =(err,req,res, next)=>{
```

Comment:
Another small thing: Please make sure to format your code and not leave any strange whitespaces (like here between the last comma and next) - if someone else wants to use your backend and uses a linting tool, they will have a file change for stuff like that. Its best practice to avoid this!